API/Models/GetQuestionDTO.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace API.Models
+{
+    public class GetQuestionDTO
+    {
+        [JsonProperty("title")] 
+        public string Title { get; set; }
+
+        [JsonProperty("aspect")] 
+        public int Aspect { get; set; }
```

Comment:
Are you sure WebApp is expecting an int here?

Reply:
Yes, as we have specified here: https://github.com/KamandaX/Wiki/pull/15
This would value would be logged along with the multiplier of an answer and sent for recommendation. Comparing integers is more efficient than strings. Also, as far as I know, we do not display aspect publicly in frontend.